lib/ansible/executor/process/worker.py

```diff
@@ -99,9 +91,6 @@ def run(self):
         # pr = cProfile.Profile()
         # pr.enable()
 
-        if HAS_ATFORK:
-            atfork()
-
```

Comment:
Looks like it re-initialized the random number generator in pycrypto after a fork.  So it definitely isn't needed if pycrypto is not installed.  i think since older paramiko could be using pycrypto, we do need to initialize it if it is installed in case we're using one of those versions of paramiko.